test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -380,19 +380,32 @@ public void TensorFlowTransformObjectDetectionTest()
         [Fact(Skip = "Model files are not available yet")]
```

Comment:
>Skip = "Model files are not available yet [](start = 14, length = 41)

do you plan unskip it in separate PR?

Reply:
I will do it as part of [https://github.com/dotnet/machinelearning/issues/904](https://github.com/dotnet/machinelearning/issues/904) in separate PR.

---
In reply to: [262684524](https://github.com/dotnet/machinelearning/pull/2855#discussion_r262684524) [](ancestors = 262684524)